lyngk_board.js

```diff
@@ -0,0 +1,210 @@
+    E1: [4, 12],
+    E2: [4, 10],
+    E3: [4, 8],
+    E4: [4, 6],
+    E5: [4, 4],
+    E6: [4, 2],
+    E7: [4, 0],
+    F1: [5, 11],
+    F2: [5, 9],
+    F3: [5, 7],
+    F4: [5, 5],
+    F5: [5, 3],
+    F6: [5, 1],
+    G1: [6, 12],
+    G2: [6, 10],
+    G3: [6, 8],
+    G4: [6, 6],
+    G5: [6, 4],
+    G6: [6, 2],
+    G7: [6, 0],
+    H1: [7, 9],
+    H2: [7, 7],
+    H3: [7, 5],
+    H4: [7, 3],
+    I1: [8, 6]
+  };
+  var nodeList = ['A1', 'B1', 'B2', 'B3', 'B4', 'C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'C7', 'D1', 'D2', 'D3', 'D4', 'D5', 'D6', 'E1', 'E2', 'E3', 'E4', 'E5', 'E6', 'E7', 'F1', 'F2', 'F3', 'F4', 'F5', 'F6', 'G1', 'G2', 'G3', 'G4', 'G5', 'G6', 'G7', 'H1', 'H2', 'H3', 'H4', 'I1'];
+  var colorValues = {
+    Ivory: "#ffffc8",
+    Blue: "blue",
+    Red: "red",
+    Green: "green",
+    Black: "black",
+    White: "gray"
+  };
+  var c=document.getElementById("lyngkboard");
+  var context=c.getContext("2d");
+  var letterToColor = {
+    'R': "Red",
+    'K': "Black",
+    'B': "Blue",
+    'I': "Ivory",
+    'G': "Green",
+    'J': "White",
+    '-': "Empty"
+  };
+  var colorArray = ['R', 'K', 'B', 'I', 'G'];
+  var kTestString1 = 'KRI|-|G|-|GR|-|IG|KJ|-|-|-|-|RK|-|J|KB|I|-|-|-|-|K|G|-|I|GBI|-|I|GIJR|B|-|RB|RKBI|G|RK|-|GB|-|-|RB|-|BK|-';
+
+  var kTestString = 'G|B|K|K|R|K|I|I|K|G|J|K|K|B|I|B|G|R|G|K|R|G|K|J|B|R|R|B|G|J|G|I|G|B|R|B|I|I|R|B|I|R|I|';
+  my.startUp = drawBoard(context);
+
+  function drawBoard(ctx) {
+    ctx.clearRect(0, 0, c.width, c.height);
+    var colorCount = {
+      Ivory: 8,
+      Blue: 8,
+      Red: 8,
+      Green: 8,
+      Black: 8,
+      White: 3
+    };
+    
+    for (var i = 0; i < verticalX.length; i++) {
+      drawLine((xRoot * verticalX[i] + padding), (yOffset * verticalY1[i] + padding), (xRoot * verticalX[i] + padding), (yOffset * verticalY2[i] + padding), ctx);
+    }
+    for (var j = 0; j < diagonalDestination.length; j++) {
+      drawLine((xRoot * diagonalSourceCoords[j][0] + padding), (yOffset * diagonalSourceCoords[j][1] + padding), (xRoot * diagonalDestination[j][0] + padding), (yOffset * diagonalDestination[j][1] + padding), ctx);
+    }
+    var boardString = document.getElementById('boardstring').innerHTML;
+    translateBoardString(boardString, context);
+    populateClaimedCanvases();
+  }
+
+  function drawLine(x1, y1, x2, y2, ctx) {
+    ctx.beginPath();
+    ctx.moveTo(x1, y1);
+    ctx.lineTo(x2, y2);
+    ctx.stroke();
+  }
+
+  function drawGamePiece(xcenter, ycenter, color, ctx) {
+    ctx.beginPath();
+    ctx.arc(xcenter, ycenter, 10, 0, 2*Math.PI);
+    ctx.fillStyle = colorValues[color];
+    ctx.fill();
+    ctx.stroke();
+  }
+
+  function drawUnclaimedPiece(color, piecePosition) {
+    var board =document.getElementById("unclaimedCanvas");
+    var ctx=board.getContext("2d");
+    ctx.beginPath();
+    ctx.arc(40, piecePosition * 62, 10, 0, 2*Math.PI);
+    ctx.fillStyle = colorValues[letterToColor[color]];
+    ctx.fill();
+    ctx.stroke();

```

Comment:
You have the act of drawing a disk at least three different times with different parameters.  Time to use a shared function.